sources/updater/updog/src/main.rs

```diff
@@ -700,8 +700,9 @@ mod tests {
 
     #[test]
     fn older_versions() {
-        // A manifest with two updates, both less than 0.1.3
-        let path = "tests/data/example_3.json";
+        // A manifest with two updates, both less than 0.1.3.
+        // Use a architecture specific JSON payload, otherwise Updog will ignore the update
```

Comment:
nit: we use lowercase when referencing our first party binaries.
```suggestion
        // Use a architecture specific JSON payload, otherwise updog will ignore the update
```

This applies to the commit message as well.

Reply:
Gotcha! Good to know 👍🏼